tests/testthat/test-license.R

```diff
@@ -0,0 +1,40 @@
+test_that("use_mit_license() works", {
+  create_local_package()
+  use_mit_license()
+
+  expect_equal(desc::desc_get("License", proj_get())[[1]], "MIT + file LICENSE")
+
+  expect_proj_file("LICENSE.md")
+  expect_true(is_build_ignored("^LICENSE\\.md$"))
+
+  expect_proj_file("LICENSE")
+  expect_false(is_build_ignored("^LICENSE$"))
+})
+
+test_that("other licenses work without error", {
+  create_local_package()
+
+  expect_error(use_agpl_license(3), NA)
```

Comment:
We have `expect_error_free()` if you want to use it.

https://github.com/r-lib/usethis/blob/master/tests/testthat/helper.R#L110-L112

Reply:
I don't really like that helper 😐